docs/testing.md

```diff
@@ -1,7 +1,7 @@
-CRI-Containerd Testing Guide
-============================
-This document assumes you have already setup the development environment (go, git, `cri-containerd` repo etc.).
- 
+CRI Testing Guide
```

Comment:
CRI Plugin

Reply:
should be ok without saying plugin here..

Reply:
but done :)